packages/tradershub/src/components/CurrencySwitcher/CurrencySwitcher.tsx

```diff
@@ -11,6 +12,8 @@ type AccountActionButtonProps = {
 };
 
 const AccountActionButton = ({ balance, isDemo }: AccountActionButtonProps) => {
+    const history = useHistory();
+    const { mutate } = useResetVirtualBalance();
```

Comment:
Shouldn't it be better to use some functional name instead of `mutate` so its easier for other developers to know which mutate function is called.?

Reply:
we can rename it in the consumer level 🙏